db/migrate/20210202052337_migrate_variant_unit_values.rb

```diff
@@ -1,16 +1,10 @@
 class MigrateVariantUnitValues < ActiveRecord::Migration
   def up
-    Spree::Variant.includes(:product).where(
-      spree_products: { variant_unit: "items" },
-      spree_variants: { unit_value: [nil, Float::NAN] }
-    ).find_each do |variant|
+    Spree::Variant.where(unit_value: [nil, Float::NAN]).find_each do |variant|
```

Comment:
lol I didn't know `NAN` existed

Reply:
I know it exists in Rails, but I'm surprised Postgres allows saving it as a value on a decimal field in the DB...

Reply:
you gotta love the :elephant: :heart: